src/translations/sqlb_ar_SA.ts

```diff
@@ -3425,69 +3514,69 @@ You can drag SQL statements from the Schema column and drop them into the SQL ed
     <message>
         <location filename="../MainWindow.ui" line="1497"/>
         <source>Ctrl+Return</source>
-        <translation></translation>
+        <translation>Ctrl+Enter</translation>
```

Comment:
Shortcuts should not be translated.
See the [wiki](https://github.com/sqlitebrowser/sqlitebrowser/wiki/Translations#dont-translate-key-names-in-shortcuts) for further information.